guacamole-common-js/src/main/webapp/modules/Client.js

```diff
@@ -1824,3 +1844,31 @@ Guacamole.Client.DefaultTransferFunction = {
     }
 
 };
+
+/**
+ * A list of possible messages that can be sent by the server and displayed to
+ * the client.
+ * 
+ * @type {!Object.<string, number>}
+ */
+Guacamole.Client.Message = {
+    
+    /**
+     * A client message that indicates that a user has joined an existing
+     * connection. This message expects a single additional argument - the
+     * name of the user who has joined the connection.
+     * 
+     * @type {!number}
+     */
+    "JOINED": 0x0001,
+    
+    /**
+     * A client message that indicates that a user has left an existing
+     * connection. This message expects a single additional argument - the
+     * name of the user who has left the connection.
+     * 
+     * @type {!number}
+     */
+    "LEFT": 0x0002
```

Comment:
I suggest renaming these to `USER_JOINED` and `USER_LEFT` to (1) align with the `GUAC_MESSAGE_USER_JOINED` and `GUAC_MESSAGE_USER_LEFT` constants and (2) clarify the context of the event.